### ISSUE TITLE
Affichage des métadonnées de la couche sélectionnée au lancement de Plume ?

### ISSUE THREAD

**Issue by alhyss** [MTES-MCT/metadata-postgresql#38]
Serait-il pertinent que, si une couche est déjà sélectionnée dans l'explorateur ou le panneau des couches au lancement de Plume, Plume affiche directement ses métadonnées ? Et est-ce faisable facilement (sinon ce n'est sans doute pas la peine de se lancer là-dedans) ?

Je me suis rendu compte que, lorsque je faisais des tests, j'avais tendance à d'abord cliquer sur la couche qui m'intéresse, puis réaliser qu'il faut lancer Plume pour voir ses métadonnées, et là je me retrouve à devoir de nouveau cliquer sur la couche... Je pense que ça vient du fait que la plupart des fonctionnalités de QGIS qui prennent pour cible un objet supposent de commencer par sélectionner l'objet, puis l'action à appliquer dans un menu ou une barre d'outils.

**Comment by alhyss** [comment]
Note que si ça devait n'être possible que pour les couches du panneau des couches (ou que pour celles de l'explorateur, mais comme c'est généralement de ce côté que c'est le plus compliqué...), ce serait déjà ça. À moins que la difficulté ne soit pas du tout au niveau de la récupération des informations sur la couche ?

**Comment by WREATCHED** [comment]
Non, le pb ne vient pas de là, mais on a dit que ce ne serait pas important pour le moment

**Comment by WREATCHED** [comment]
**Réponse en video**
https://user-images.githubusercontent.com/66324136/161994169-efa92ea9-54e1-4947-8225-d753693582ba.mp4

**Comment by alhyss** [comment]
Exactement ce que j'avais en tête ! Merci infiniment au nom de tous ceux qui cliquent d'abord sur la cible et réfléchissent ensuite :)

**Comment by WREATCHED** [comment]
**En complément, la dernière cerise sur le gâteau**
https://user-images.githubusercontent.com/66324136/162023585-f25c34db-365c-42d1-971f-5390188744ff.mp4
